build.gradle

```diff
@@ -10,10 +10,10 @@ plugins {
 	id "idea"
 	id "maven-publish"
 	id "org.quiltmc.loom" version "1.0.+" apply false
-	id "org.quiltmc.gradle.licenser" version "1.1.2"
-	id "net.kyori.indra.git" version "3.0.0"
+	id "org.quiltmc.gradle.licenser" version "1.+"
+	id "net.kyori.indra.git" version "3.0.1"
 	id "com.matthewprenger.cursegradle" version "1.4.0"
-	id "com.modrinth.minotaur" version "2.4.4"
+	id "com.modrinth.minotaur" version "2.+"
```

Comment:
but please don't do this; there was a breaking change done in a 2.x update iirc

Reply:
Undoed.